VERSION

```diff
@@ -1 +1 @@
-1.3.12
+2.0.0-alpha
```

Comment:
We want to version this to alpha?

Reply:
I was just thinking once we know it is working we can put it to 2.0.0